packages/orbit-tailwind-preset/README.md

```diff
@@ -0,0 +1,52 @@
+# Orbit tailwind preset
+
+This package contains the Orbit Tailwind presets.
+
+Currently, there are two presets available:
+
+- `orbitPreset` - Orbit's foundation styles, including typography, colors, spacing, etc.
```

Comment:
Should we mention this is the default export? Or maybe we can decide not to have default exports, so all exports are named and explicitly imported 💭

Reply:
hm, inside usage there is an example that's showing it as default

Reply:
Yes, I saw it. But it happened to me in the past 😄 when trying to use this on another repository I was trying to import this name just to later find out that I should import it as default. I think something as simple as this should be enough and clarifying
```suggestion
- `orbitPreset` **(default export)** - Orbit's foundation styles, including typography, colors, spacing, etc.
```